indico/modules/events/contributions/blueprint.py

```diff
@@ -24,3 +25,11 @@
                       virtual_template_folder='events/contributions', url_prefix='/event/<confId>')
 
 _bp.add_url_rule('/manage/contributions/', 'manage_contributions', RHManageContributions)
+_bp.add_url_rule('/manage/contributions/create',
+                 'manage_create_contrib', RHManageContributionCreate, methods=('GET', 'POST'))
+
+# Single contribution
+_bp.add_url_rule('/manage/contributions/<int:contrib_id>/edit',
+                 'manage_update_contrib', RHManageContributionUpdate, methods=('GET', 'POST'))
+_bp.add_url_rule('/manage/contributions/<int:contrib_id>/delete',
```

Comment:
without the `/delete` at the end of the URL (sending a DELETE request to `.../delete` is a bit pointless)